README.md

```diff
@@ -321,7 +322,8 @@ Currently supported distributions are:
 - [vmctl](https://github.com/VictoriaMetrics/vmctl)
 - [xh](https://github.com/ducaale/xh)
 - [yh](https://github.com/andreazorzetto/yh)
-- [yj](https://github.com/sclevine/yj)
+- [yj](https://github.com/sclevine/yj)/
```

Comment:
hey @patsevanton ; I guess there is a small typo here

Reply:
Fixed